package.json

```diff
@@ -110,7 +111,7 @@
     ]
   },
   "engines": {
-    "node": ">=12.18.0 <15.x",
+    "node": ">=10.16.0 <17.x",
```

Comment:
Why?

Reply:
@kilemensi This is on me. Constantly switching node versions caused this. Changed it